cw_bitcoin/lib/electrum_wallet_addresses.dart

```diff
@@ -53,41 +51,71 @@ abstract class ElectrumWalletAddressesBase extends WalletAddresses with Store {
   @override
   @computed
   String get address {
-    if (receiveAddresses.isEmpty) {
-      final address = generateNewAddress().address;
-      return walletInfo.type == WalletType.bitcoinCash ? toCashAddr(address) : address;
-    }
-    final receiveAddress = receiveAddresses.first.address;
+    if (!isEnabledAutoGenerateSubaddress) {
```

Comment:
I believe this is wrong, shouldn't this be without the negation (i.e without `!`)?
because if auto generate is enabled, then we get the first unused address
else we get the first address or the previous address record

Reply:
fixed